src/stdx/prng.zig

```diff
@@ -503,7 +503,7 @@ test Combination {
 }
 
 /// An iterator style API for selecting a single element out of the given weighted sequence,
-/// without apriory knowledge about the total weight.
+/// without a priory knowledge about the total weight.
```

Comment:
```suggestion
/// without a priori knowledge about the total weight.
```

Reply:
I even looked up this in wikipedia and still messed it up, thanks!